pandas/tests/tseries/offsets/test_offsets_properties.py

```diff
@@ -15,6 +15,7 @@
 import pytest
 
 import pandas as pd
+from pandas import Timestamp
```

Comment:
Should these just stay `pd.` since already imported or did you want to remove the preceding import?

Reply:
In a later step I'm looking to separate out the remaining `pd.*` usages into a separate not-just-tslibs test module